test/unittests/test_recordstore.py

```diff
@@ -457,6 +461,9 @@ def test_create_project(self):
     def test_project_info(self):
         self.assertEqual(self.store.project_info("TestProject")["name"], "TestProject")
 
+    def test_clear(self):
+        pass  # raises UserWarning
```

Comment:
I have fetched it but haven't merged it and am not planning to, so I believe it should be fine to amend it. (If this messes something up I can always remove and re-add the remote apdavison.)